src/backend/marsha/core/utils/api_utils.py

```diff
@@ -49,3 +50,8 @@ def generate_hash(secret, message):
     return hmac.new(
         secret.encode("utf-8"), msg=message, digestmod=hashlib.sha256
     ).hexdigest()
+
+
+def generate_salted_hmac(secret, key):
```

Comment:
Why not reusing the function `generate_hash` ?

Reply:
I got errors trying to use it TypeError: object supporting the buffer API required and thought it would be too long for an URL so I didn't insist and wanted to make a simpler one